docs/api/ethereum.md

```diff
@@ -1439,6 +1443,10 @@ const mutationId = '0x767857798ea7c8d21ad72c4d7e054ed45ab5d177c06baa2183dbfc3b61
 const mutation = new Mutation(provider, mutationId);
 ```
 
+### cancel()
+
+An `asynchronous` class instance `function` which attempts to cancel a mutation. You can only cancel a mutation that has not yet been accepted onto the blockchain. Canceling works by overwriting a pending mutation (using the same [nonce](https://kb.myetherwallet.com/en/transactions/what-is-nonce/)) with a new mutation that performs no action, which could also lead the `cancel` function to fail. The `cancel` function will throw an error if the mutation has already been accepted onto the blockchain or if you are not the originator of the mutation you want to cancel.
+
 ### complete()
 
 An `asynchronous` class instance `function` which waits until the mutation reaches the specified number of confirmations.
```

Comment:
An `asynchronous` class instance `function` which waits until a mutation reaches a specified number of confirmations.